openwisp_users/widgets.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.admin.widgets import AutocompleteSelect as BaseAutocompleteSelect
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+
+class OrganizationAutocompleteSelect(BaseAutocompleteSelect):
+    class Media:
+        js = ['admin/js/jquery.init.js', 'openwisp-users/js/org-autocomplete.js']
+
+    def get_url(self):
+        return reverse('admin:ow-auto-filter')
+
+    def optgroups(self, name, value, attrs=None):
+        groups = super().optgroups(name, value, attrs)
+        if value == [''] and len(groups[0][1]) == 1:
+            groups[0][1].append(
+                self.create_option(
+                    name=name,
+                    value='null',
+                    label=_('Shared systemwide (no organization)'),
```

Comment:
Let's avoid duplicating the definition of this string which we are using in multiple places, can you please define it in a constant somewhere (eg: multitenancy.py) and change the rest of the code to use the constant? This way we ensure that any change to this value in the future will remain constant across the entire application.